diagnostic.md

```diff
@@ -51,25 +57,25 @@ Why do our `model` files appear to be lacking so much code, when they're doing
 so much for us?
 
 ```md
-// your response here
+They inherit from Application Record.
 ```
 
 What does typing `bin/rails dbconsole` in a directory that houses a rails app do for
 us?
 
 ```md
-// your response here
+It opens up the pry command line.
 ```
 
 What does typing `rails db` in a directory that houses a rails app do for us?
 
 ```md
-// your response here
+The same thing?
```

Comment:
Sort of, but this will actually get you into the underlying database, in this case psql